app/src/main/java/com/andrew/tasky/presentation/agenda_item_detail/DatePickerFragment.kt

```diff
@@ -1,32 +1,31 @@
-package com.andrew.tasky.presentation.task_detail
+package com.andrew.tasky.presentation.agenda_item_detail
 
 import android.app.DatePickerDialog
 import android.app.Dialog
 import android.os.Bundle
 import android.widget.DatePicker
 import androidx.fragment.app.DialogFragment
 import androidx.fragment.app.setFragmentResult
-import java.text.SimpleDateFormat
-import java.util.*
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
 
 class DatePickerFragment: DialogFragment(), DatePickerDialog.OnDateSetListener {
 
-    private val calendar = Calendar.getInstance()
+    private val currentDateTime = LocalDateTime.now()
 
     override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
-        val year = calendar.get(Calendar.YEAR)
-        val month = calendar.get(Calendar.MONTH)
-        val day = calendar.get(Calendar.DAY_OF_MONTH)
+        val year = currentDateTime.year
+        val month = currentDateTime.monthValue-1
```

Comment:
What's the reason for `-1`? 
For these things I personally like to add a comment explaining why, since even if you're working on this project alone, you'll ask yourself why you subtracted 1 here if you come back to this after a few weeks.

Generally, whenever someone could ask themselves "Why is this happening?" a comment is great. If you add a comment that explains WHAT happens, it's usually a sign that you could improve the readability of the code to get rid of the comment :)